packages/override_api_endpoint/pubspec.yaml

```diff
@@ -0,0 +1,24 @@
+name: override_api_endpoint
+version: 1.0.0
+homepage: https://github.com/leancodepl/flutter_corelibrary/tree/master/packages/override_api_endpoint
+repository: https://github.com/leancodepl/flutter_corelibrary
+description: >-
+  This package is prepared so that backend developers can override api endpoint
+  fo some local one without the need to compiling the app. Desired behavior is
+  realized with deeplinks. The override url is persisted until new deeplink
+  is used.
+
+environment:
+  sdk: '>=2.12.0 <3.0.0'
+  flutter: '>=2.0.0'
+
+dependencies:
+  shared_preferences: ^2.0.6
+  uni_links: ^0.5.1
+
+dev_dependencies:
+  build_runner: ^1.11.5
```

Comment:
There are no tests right now so `build_runner`, `coverage`, `mockito`, and `test` are never used.